lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -39,6 +39,25 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
         this.logger.verbose("initializeAuthorizationRequest called", request.correlationId);
         const generatedPkceParams = await this.browserCrypto.generatePkceCodes();
 
+        // Generate Session Transport Key if Refresh Token Binding is enabled
+        if (this.config.system.refreshTokenBinding) {
+            this.logger.verbose("Refresh token binding enabled, attempting to generate Session Transport Key");
+            try {
+                const sessionTransportKeyThumbprint = await this.browserCrypto.getPublicKeyThumbprint(request, CryptoKeyTypes.StkJwk);
+                request.stkJwk = sessionTransportKeyThumbprint;
+                this.logger.verbose("Successfully generated and stored Session Transport Key");
+            } catch(error) {
+                if(error instanceof BrowserAuthError) {
+                    if (error.errorCode === "key_generation_failed") {
```

Comment:
If you stick with this, can you compare against `BrowserAuthErrorMessage.keyGenerationFailed.code`?

Reply:
Updated to access the code string programatically